### ISSUE TITLE
app's style override.

### ISSUE THREAD

**Issue by Truerall** [box/box-android-browse-sdk#6]
Hi, I have unusual trouble with usage of this sdk.
After adding this sdk as dependency to my app like this(Im using Android Studo v1.2.2):

 compile 'com.box:box-android-browse-sdk:1.1.0'

even without usage of sdk classes in my app, Im getting changes to some of my view styles.
for example

before
![screenshot_1](https://cloud.githubusercontent.com/assets/9592676/8522209/42cf272c-23f4-11e5-892d-e3b2ccd6aacd.png)

after

![screenshot_2](https://cloud.githubusercontent.com/assets/9592676/8522211/485ee3c6-23f4-11e5-8b60-e905d3fd196b.png)

Any suggestion how to fix this? (also I tried to add your sdk as library project, and disable all your styles, but this did not helped)

It is not a problem to implement box api using your content sdk, but it much more comfortable to use your picker in some cases.

**Comment by doncung** [comment]
Can you try your code with just importing Android's app compat libraries instead of the whole browse sdk. 
 
Just put into your build.gradle:
compile 'com.android.support:appcompat-v7:22.1.1'
compile 'com.android.support:recyclerview-v7:22.1.1'

We are using these libraries for better backwards compatibility, but it might have the unintended side effect of changing your default font style.

**Comment by Truerall** [comment]
Thanx for your help. I already have this dependencies in build.gradle, but with elder version :21.0.3

compile 'com.android.support:appcompat-v7:21.0.3'
compile 'com.android.support:recyclerview-v7:21.0.3'

As far as my app's theme is extended from AppCompat 
<style name="AppTheme" parent="@style/Theme.AppCompat.NoActionBar">

Im making a conclusion that google made some visual changes for this theme. for example attribute <item name="android:textAllCaps"></item> was changed to true by default.

I didnt knew that gradle chooses the newest library version comparing the nested dependenceis.
It is obvious now that it took latest AppCompat version from your build.gradle(22.1.1).

Sorry for wasting your time. And thank you for help and quick answer!

P.S.
I have found that some methods that shown in documentation are deprecated. There are links to full content sdk instead of browse-sdk.
